release/models/segment-routing/openconfig-segment-routing.yang

```diff
@@ -687,6 +752,33 @@ module openconfig-segment-routing {
     }
   }
 
+  grouping sr-igp-interface-flex-algo-prefix-sid-config {
+    description
+      "Configuration parameters relating to an IGP prefix SID advertisement for
+       Flexible Algorithms";
+    leaf prefix {
+      type inet:ip-prefix;
+      description
+        "The IP prefix for which the IGP prefix SID should be advertised. The
+         value specified is a local prefix on the interface which is advertised
+         into the IGP.";
+    }
+    leaf flex-algo-name {
+      type leafref {
+        path "/network-instances/network-instance/"
+           + "flex-algorithms/algorithm/name";
+      }
```

Comment:
again, I would completely git rid of any algo name in the model, we already have unique algo number.